.travis.yml

```diff
@@ -18,14 +18,12 @@ matrix:
     - php: 5.5
       env: COMPOSER_FLAGS="--prefer-lowest" SYMFONY_DEPRECATIONS_HELPER=weak
     - php: 5.6
-    # Symfony 2 LTS
-    - php: 7.0
-      env: SYMFONY_LTS='^2'
     # Symfony 3 LTS
     - php: 7.0
       env: SYMFONY_LTS='^3'
     # Symfony 4
     - php: 7.1
+      env: SYMFONY_PHPUNIT_VERSION='6.5'
```

Comment:
For some reason PHPUnit 5.7 triggered deprecation on this job (see https://travis-ci.org/FriendsOfSymfony/FOSRestBundle/jobs/474149446). But since we can use PHPUnit 6.5 anyway I suggest not to spend time on investigating this now (we *could* do that later though).